### PR TITLE
Rename Alex's Post, add a new post related to the Gap Anlaysis and ad…

### PR DESCRIPTION
**Pull request by alebre**
…d stupid README

### CODE REVIEW

File: _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+With the next [OpenStack PTG](https://www.openstack.org/ptg/) in mind,
+we conducted a first gap analysis of the OpenStack code base with
+respect to the Edge Computing Challenges.  The outputs of this
+brainstorming session, which has been organized with two folks from
+Ericsson, are:
+* The identification and classification of a couple of
+  features we believe edge administrators/developers may
+  expect. Concretely, we define 7 levels of features, from the simplest
+  such as being able to start a VM on a specific location, to the most
+  advances ones, including the interoperability of different cloud
```
Comment (rcherrueau) [inline _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown]: advanced

File: _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  stacks (OpenStack/Kubernetes/...).
+* An analysis of different deployment scenarios the OpenStack code
+  base provides (cells, regions, federations...)  w.r.t the
```
Comment (rcherrueau) [inline _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown]: federation ...

File: _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  aforementioned levels.
+
+Obviously, this study is an ongoing action that should be refined with
+the expertise of additional folks but at least we started it !
```
Comment (rcherrueau) [inline _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown]: folks, but

Comment (rcherrueau) [inline _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown]: started it!

**Review by rcherrueau — CHANGES_REQUESTED** [review]
some typo

File: _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown (hunk 2 above)
Comment (alebre) [reply _posts/2018-01-22-OpenStack-Edge-Gap-Analysis.markdown]: we puts s at regions cells. you can have several federations.

**Review by rcherrueau — APPROVED** [review]
👍

**Pull request merged**